lib/gh_events.ts

```diff
@@ -37,21 +37,25 @@ interface Comment {
   updated_at: string;
 }
 
-interface PullRequest {
+export interface PullRequest {
   html_url: string;
   user: Actor;
   title: string;
   body: string;
   number: number;
-  labels: string[];
+  labels: string[] | { name: string }[];
```

Comment:
Why was this changed? Let's keep either `string[]` or `{ name: string }[]` only. This could lead to confusion.